utility/subcommands.py

```diff
@@ -0,0 +1,264 @@
+"""
```

Comment:
Any reason why we named this file as `subcommands`? IMO, it does not match with the purpose or objective of the tasks performed.